GeometryService/inc/STMMaker.hh

```diff
@@ -54,6 +54,9 @@ namespace mu2e {
     double       _magnetHoleHalfHeight;
     std::string  _magnetMaterial;
     double       _magnetField;
+    double       _magnetFieldx;
```

Comment:
Should we be adding magnetic fields not through the field maps normally? Is this for testing purposes?